frontend/nuxt.config.ts

```diff
@@ -191,160 +145,20 @@ const config: NuxtConfig = {
-  },
-  sitemap: {
-    hostname: "https://openverse.org",
-    i18n: {
-      locales: openverseLocales.map((l) => l.iso),
-      routesNameSeparator: "___",
-    },
-  },
-  /**
-   * Map the old route for /photos/_id page to /image/_id permanently to keep links working.
-   * See the redirect module for more info.
-   * {@link https://github.com/nuxt-community/redirect-module#usage}
-   */
-  redirect: {
-    rules: [
-      { from: "^/photos/(.*)$", to: "/image/$1", statusCode: 301 },
-      { from: "/meta-search", to: "/about", statusCode: 301 },
-      { from: "/external-sources", to: "/about", statusCode: 301 },
-    ],
-    // If the URL cannot be decoded, we call next() to show the client-side error page.
-    onDecodeError: (
-      _error: Error,
-      _req: IncomingMessage,
-      _res: http.ServerResponse,
-      next: NextFunction
-    ) => {
-      return next()
-    },
-  },
-  sentry: sentryConfig,
-  build: {
-    templates: [
-      {
-        src: "./nuxt-template-overrides/App.js",
-        dst: "App.js",
-      },
-      {
-        src: "./nuxt-template-overrides/index.js",
-        dst: "index.js",
-      },
-    ],
-    filenames,
-    friendlyErrors: false,
-    postcss: {
-      postcssOptions: {
-        preset: {
-          features: {
-            // Disable conversion of logical properties to physical properties
-            // e.g.: `margin-inline-start` is NOT converted to `margin-left`
-            // Necessary for RTL support.
-            "logical-properties-and-values": false,
-          },
-        },
-        plugins: {
-          tailwindcss: {
-            config: path.resolve(__dirname, "tailwind.config.ts"),
-          },
-          "postcss-focus-visible": {},
-        },
-      },
-    },
-    extend(config, ctx) {
-      // Enables use of IDE debuggers
-      config.devtool = ctx.isClient ? "source-map" : "inline-source-map"
-    },
-    transpile: [({ isLegacy }) => (isLegacy ? "axios" : undefined)],
-  },
-  typescript: {
-    typeCheck: {
-      typescript: {
-        configFile: "./tsconfig.json",
-        extensions: {
-          vue: true,
-        },
-      },
-    },
-  },
-  storybook: {
-    port: 6006,
-    addons: [
-      {
-        name: "@storybook/addon-essentials",
-        options: {
-          backgrounds: true,
-          viewport: true,
-          toolbars: true,
-        },
-      },
-    ],
-    parameters: {
-      backgrounds: {
-        default: "White",
-        values: [
-          { name: "White", value: "#ffffff" },
-          { name: "Dark charcoal", value: "#30272e" },
-        ],
-      },
-      options: {
-        storySort: {
-          order: ["Introduction", ["Openverse UI"], "Meta"],
-        },
-      },
-      viewport: {
-        viewports: VIEWPORTS,
-      },
-    },
+    trailingSlash: false,
+    vueI18n: "./src/vue-i18n",
   },
   plausible: {
+    enabled: !isTest,
+    logIgnoredEvents: !isProductionBuild,
     trackLocalhost: !isProdNotPlaywright,
```

Comment:
```suggestion
    ignoredHostnames: ["localhost", "staging.openverse.org"]
```

It looks like this `trackLocalhost` field is deprecated now for a new one:

```
ignoredHostnames?: string[] | undefined
Hostnames to ignore when tracking events.

@default

['localhost']
```